ewkcoffea/modules/selection_wwz.py

```diff
@@ -175,6 +176,39 @@
         "EGamma"         : dataset_dict["2018"]["DoubleMuon"],
         "MuonEG"         : dataset_dict["2018"]["DoubleMuon"] + dataset_dict["2018"]["EGamma"],
     },
+    "B": {
```

Comment:
If we are going to have the trigger list and exclude dict here, it should be removed from `selection_run3_2lep.py` (we definitely do not want to store it in two places). The other option is to add it as a json that both can reference. However, jsons do not allow comments and some of the things here we really want comments for (e.g. the Era C complications).

So I would suggest leaving the dictionary here (and then just removing it from the dilepton section file). The di lepton processor can just point to this copy here.